projects/1.18/assets/crossroads-mc/crossroads/lang/zh_cn.json

```
@@ -1,861 +1,1086 @@
+	"item.crossroads.beam_cage": "光束笼",
+	"item.crossroads.prototype_pistol": "原型手枪",
+	"block.crossroads.molten_copshowium": "熔融锞源",
+	"item.crossroads.prototype_watch": "原型怀表",
+	"block.crossroads.mechanical_arm": "机械臂",
+	"block.crossroads.redstone_registry": "红石注册表",
+	"item.crossroads.adamant": "精金",
+	"item.crossroads.sulfur": "硫",
+	"item.crossroads.vanadium_oxide": "氧化钒",
+	"item.crossroads.phil_stone": "思索者之石",
+	"item.crossroads.prac_stone": "实践者之石",
+	"item.crossroads.alch_crystal": "炼金术水晶",
+	"item.crossroads.waste_salt": "炼金术盐",
+	"block.crossroads.alch_tube": "玻璃导管",
+	"block.crossroads.crystal_alch_tube": "水晶导管",
+	"block.crossroads.fluid_injector": "玻璃流体注入器",
+	"block.crossroads.crystal_fluid_injector": "水晶流体注入器",
+	"block.crossroads.flow_limiter": "玻璃限流导管",
+	"block.crossroads.crystal_flow_limiter": "水晶限流导管",
+	"block.crossroads.heated_tube": "玻璃加热导管",
+	"block.crossroads.crystal_heated_tube": "水晶加热导管",
+	"block.crossroads.cooling_coil": "玻璃冷却螺旋管",
+	"block.crossroads.crystal_cooling_coil": "水晶冷却螺旋管",
+	"block.crossroads.chemical_vent": "化学品排放口",
+	"block.crossroads.reaction_chamber": "玻璃反应容器",
+	"block.crossroads.crystal_reaction_chamber": "水晶反应容器",
+	"item.crossroads.liech_wrench": "列支敦斯登海军扳手",
+	"item.crossroads.leyden_jar": "莱顿瓶",
+	"block.crossroads.tesla_coil": "特斯拉线圈",
+	"block.crossroads.tesla_coil_top_normal": "特斯拉线圈顶部",
+	"block.crossroads.tesla_coil_top_distance": "长距离特斯拉线圈顶部",
+	"block.crossroads.tesla_coil_top_intensity": "高强度特斯拉线圈顶部",
+	"block.crossroads.tesla_coil_top_attack": "攻击性特斯拉线圈顶部",
+	"block.crossroads.tesla_coil_top_efficiency": "高效率特斯拉线圈顶部",
+	"block.crossroads.tesla_coil_top_decorative": "装饰性特斯拉线圈顶部",
+	"block.crossroads.heat_limiter_basic": "温度调节器",
+	"block.crossroads.heat_limiter": "红石温度调节器",
+	"block.crossroads.dynamo": "发电机",
+	"block.crossroads.reagent_tank": "玻璃存储容器",
+	"block.crossroads.crystal_reagent_tank": "水晶存储容器",
+	"block.crossroads.reagent_pump": "玻璃泵",
+	"block.crossroads.crystal_reagent_pump": "水晶泵",
+	"block.crossroads.maxwell_demon": "麦克斯韦妖",
+	"item.crossroads.phial": "试管",
+	"item.crossroads.phial_glass": "玻璃试管",
+	"item.crossroads.phial_cryst": "水晶试管",
+	"block.crossroads.phial_holder": "试管架",
+	"item.crossroads.florence_flask_glass": "玻璃平底烧瓶",
+	"item.crossroads.florence_flask_cryst": "水晶平底烧瓶",
+	"item.crossroads.shell_glass": "玻璃壳",
+	"item.crossroads.shell_cryst": "水晶壳",
+	"item.crossroads.nitroglycerin": "瓶装硝化甘油",
+	"item.crossroads.poison_vodka": "毒伏特加",
+	"item.crossroads.double_poison_vodka": "双倍毒伏特加",
+	"block.crossroads.reds_alch_tube": "玻璃红石导管",
+	"block.crossroads.crystal_reds_alch_tube": "水晶红石导管",
+	"block.crossroads.glassware_holder": "玻璃器皿架",
+	"block.crossroads.alembic": "蒸馏器",
+	"item.crossroads.solid_quicksilver": "固态汞",
+	"item.crossroads.custom_material": "晶化炼金块",
+	"block.crossroads.alchemy_chart": "炼金术图表",
+	"block.crossroads.densus_plate": "密素板",
+	"block.crossroads.anti_densus_plate": "反密素板",
+	"item.crossroads.solid_eldrine": "凝固奥秘素",
```

```suggestion
	"item.crossroads.solid_eldrine": "固化奥秘素",
```
固化的话，可以说氮的固化、水泥的固化。而凝固只能用于液体，水的凝固。所以这里的奥秘素之类的我觉得更像气体一点，所以使用固化一词。

---

固化更多表达一个过程，现在是描述其性状
之前其实也是固化，后来我觉得不对才改的

---

也对